.github/lighthouserc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "ci": {
+    "numberOfRuns": 3,
+    "settings": {
+      "preset": "mobile"
```

Comment:
I like that you added a lighthouse config + CI test :+1: 

But I don't think we need to worry about `mobile` as much for this website. It is decidedly _not_ mobile-first :sweat_smile:

Reply:
Ah that's true! I'll change it to `"desktop`" since that's what we actually want to test.